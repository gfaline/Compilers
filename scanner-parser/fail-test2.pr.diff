diff: fail-test2.pr.err: No such file or directory
diff: testss/tests/fail-test2.pr.err: No such file or directory
