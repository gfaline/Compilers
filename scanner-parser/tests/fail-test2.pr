
objdef Student
{
  int id;
  str name;
}


Student stu;
stu.id = 25111;
stu.name = 'John';
fn celebrate(obj o)
{
  if o.name == 'John'
 {
     print('Time to buy my own insurance...')
  }
}

bind(stu.name, celebrate(stu));