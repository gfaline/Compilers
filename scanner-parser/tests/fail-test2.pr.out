tests/fail-test2.pr
